packages/design-system/src/Documentation/components/ColorBlock.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
Just curious - why did you roll your own component instead of using [storybook's color palette doc block](https://storybook.js.org/docs/react/api/doc-block-colorpalette)?

Reply:
Was not aware of this... will update this later.